cla_backend/apps/call_centre/views.py

```diff
@@ -15,7 +15,7 @@
 from legalaid.permissions import IsManagerOrMePermission
 
 from rest_framework import viewsets, mixins, status
-from rest_framework.decorators import action, link
+from rest_framework.decorators import detail_route, list_route
```

Comment:
can we remove @list_route as we don't use it?

Reply:
It's used lowered down in this file https://github.com/ministryofjustice/cla_backend/blob/feature/LGA-1772-drf-2.4.8/cla_backend/apps/call_centre/views.py#L265

This was actually already a `@list_route` which was being imported from `from core.drf.decorators import list_route`